docs/_data/portland-2024-config.yaml

```diff
@@ -107,23 +107,23 @@ date: # how do we handle these? human readable would be nice.
 
   # Conference is special for the index to show the combined conference days
   conference:
-    event: Main Conference
+    event: Speaker talks
     date: April 15-16
-    summary: The main days of the conference. We will be running one main track of talks, as well as a <a href="/conf/portland/2024/job-fair/">Sponsor Expo</a>, <a href="/conf/portland/2024/lightning-talks/">Lightning Talks</a> & <a href="/conf/portland/2024/unconference/">Unconference</a>.
+    summary: The main days of the conference. Listen to a diverse panel of speakers share their insights and experience. Currently accepting Call for Proposals.
```

Comment:
```suggestion
    summary: The main days of the conference. Our speakers share their insights and experience, Other events include unconference discussions, lighting talks and job fair. 
```

Reply:
This is copy from Katie, I think she wanted this stuff to change across time, which is definitely a bigger lift, but probably useful? 🤷

Reply:
Yeah, but unless we're systematic about it, we're gonna drop stuff.

Reply:
I think we can just leave it off, and ditch most of my edits here as well as we're adding further stuff below

Reply:
Actually, we could just wrap this in the conditional, although it's some pretty hairy nesting of syntax... :-D